### PR TITLE
Alex/add pictures to guide and landing

### PR DESCRIPTION
**Pull request by FluffyPenguin**
# Description

Adds screenshot to the landing page and a quick guide on setting up the webhook in google ads.
image
image

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: frontend/src/app/components/crm/guide/guide.component.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+* {
+    box-sizing: border-box;
+  }
```
Comment (r0ddy) [inline frontend/src/app/components/crm/guide/guide.component.css:3]: Delete tab here.

File: frontend/src/app/components/crm/guide/guide.component.html [hunk 2]
```diff
@@ -1 +1,25 @@
...
+                <li>Click save at the bottom of the Form page.</li>
+                <li>Congratulations! Your form is ready to delivery leads for you to view here! As soon as you receive a new lead, you will be able to view it on the "Leads" page!</li>
+            </ol>
+        </div>
+        <div>
+            <img id="screenshot" src="assets/imgs/settingUpUserJourney.png">
+        </div>
+    </div>
+    
+
```
Comment (r0ddy) [inline frontend/src/app/components/crm/guide/guide.component.html:24]: Delete these new lines.

File: frontend/src/app/components/crm/guide/guide.component.html [hunk 3]
```diff
@@ -1 +1,25 @@
-<p>guide works!</p>
+<div id="content" fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="57px">
+    <div id="settingUp" class="guideModule" fxLayout="row" fxLayoutAlign="space-between start" fxLayoutGap="100px">
+        <div id="settingUpText">
+            <span class="mat-h1">
+                Setting Up:
+            </span>
+            <ol>
+                <li>At the top right of the page, click on the User icon.</li>
+                <li>Your unique webhook and google key should appear, copy them to the Google Form Ad(s) that you wish to link to miniCRM.
```
Comment (r0ddy) [inline frontend/src/app/components/crm/guide/guide.component.html:9]: Capitalize `google` in `google key` and change `webhook` to `webhook url`.

Comment (FluffyPenguin) [reply frontend/src/app/components/crm/guide/guide.component.html:9]: fixed

**Pull request merged**